knime_extension/src/nodes/spatialmodels.py

```diff
@@ -0,0 +1,1219 @@
+import knime_extension as knext
+import util.knime_utils as knut
+import pandas as pd
+import geopandas as gp
+from mgwr.gwr import GWR, MGWR
```

Comment:
also here, maybe move some of the imports into the respective execute methods to speed up loading the geospatial extension